.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  env: {
+    es6: true,
+    node: true,
```

Comment:
Do we need `node`?

Reply:
I'm not 100% sure - if we happen to have tests that use any of the node tools it might be good to have that.